src/extension.ts

```diff
@@ -424,3 +423,33 @@ export async function activate(context: vscode.ExtensionContext): Promise<void>
         context.subscriptions.push(createClient(dirToAnalyze).start());
     }
 }
+
+// Search all the workspace folders and return the first that contains .phan/config.php
+// This will return the folder name in an array as required, and just only one folder,
+// the first met. We can easy modify the function to add all valid workspace folders to the array.
+function findValidProjectDirectories(): string[] {
+    // Get the fsPath(file system path) of all workspace folders.
+    const VSCodeFolders = vscode.workspace.workspaceFolders;
```

Comment:
nit: vscodeFolders would be more consistent with other names - uppercase looks like a class (e.g. `import { DocumentFilter, RelativePattern } from 'vscode';`)

Reply:
Yeah, I will change it to "vscodeFolders" if "vscode" is a standard